blathering-barnacles/Bangazon-2
# Jb product detail

**Description:**

Added the ability to view details of a single product. Steps completed: Added a URL path that takes to user to that products detail page using the product id in the url. A template that displays the product name, price, description and quantity available. (Right now that is just the original amount available, not current inventory. I broke that out into another ticket which I will do next.) A view that includes the productDetail function which queries the db for product details and renders the template. I also included a button that will eventually allow the user to add the product to their shopping cart if they're logged in.

**Steps to Test:**

Checkout to my branch jb-product-detail
go to localhost:8000/products/1 (or the id of any products you have in your database)
You should see the Name, description, price and quantity of the product.

Link to Feature Ticket: #36
